src/components/lobby/LobbySeeks.tsx

```diff
@@ -1,15 +1,77 @@
+// tslint:disable:no-console
 import * as React from 'react';
-
-class LobbySeeksComponent extends React.Component<any, any> {
+import styled from 'styled-components';
+import {CreatedGames} from '../../store/MockData';
+const Container = styled.div `
+grid-column-start: 2;
+grid-column-end: 3;
+grid-row-start: 2;
+grid-row-end: 3;
+background-color: white;
+opacity: 0.8;
+height: auto;
+width: auto;
+overflow-y: scroll;
+padding: 50px 40px 0px 40px;
+box-shadow: 5px 5px 30px black, -5px -3px 30px black;
+  
+`
+const ListItemMatch = styled.div `
+display: grid;
+grid-template-columns: 20% 20% 20% 20% 20%;
+border-bottom: 1px solid black;
```

Comment:
Can I suggest adding a `last-of-type` on this ListItemMatch on the bottom of this to remove the border-bottom on the last list item. View example below in the new comment below.